scripts/partition_manager.py

```diff
@@ -565,22 +614,42 @@ def parse_args():
     parser.add_argument("--input-files", required=True, type=str, nargs="+",
                         help="List of paths to input yaml files. ")
 
-    parser.add_argument("--flash-size", required=True, type=int,
-                        help="Flash size of chip in kB.")
+    parser.add_argument("--output-partitions", required=True, type=str,
+                        help="Path to output partition configuration file.")
+
+    parser.add_argument("--output-regions", required=True, type=str,
+                        help="Path to output regions configuration file.")
 
-    parser.add_argument("--flash-start", type=lambda x: int(x, 0), default=0,
-                        help="Start address of flash.")
+    parser.add_argument("--static-config", required=False, type=argparse.FileType(mode='r'),
+                        help="Path static configuration.")
 
-    parser.add_argument("--output", required=True, type=str,
-                        help="Path to output file.")
+    parser.add_argument("--regions", required=False, type=str, nargs='*',
+                        help="Space separated list of regions. For each region specified here, one must specify"
+                             "--{region_name}-base-addr and --{region_name}-size. If the region is associated"
+                             "with a driver, the device name must be given in --{region_name}-device (e.g. an "
+                             "external flash driver. For regions with placement strategy 'complex' it is possible"
```

Comment:
Why only for "complex" placement strategy?

Reply:
This feature is only used when renaming sub images in multi domain builds, which is not yet supported. Otherwise it is sufficient to name the unused area after the region IMO.

Reply:
Ok, I guess it makes more sense when the multi-domain renaming is added.